exasol_sagemaker_extension/autopilot_handler.py

```diff
@@ -44,4 +48,23 @@ def check_training_status(job_name: str):
         describe_response["AutoMLJobSecondaryStatus"]
 
 
+def deploy_endpoint(
+        job_name: str, instance_type: str, instance_count: int):
+
+    unique_endpoint_name = job_name  # endpoint_name = job_name
+    automl = AutoML.attach(auto_ml_job_name=job_name)
+    best_candidate = automl.describe_auto_ml_job(job_name)["BestCandidate"]
+    automl.deploy(
+        initial_instance_count=instance_count,
+        instance_type=instance_type,
+        candidate=best_candidate,
+        wait=True,
+        endpoint_name=unique_endpoint_name
```

Comment:
I am still not sure, if this is a good idea. I can understand, when we don't extend the model name with the time during training, but for the endpoint we probably can extend the name to avoid any collision. I would add the candidate name and the time. The first one, avoids problems if we allow in the future the deployment of different candidates, the second one helps with collision caused by manually created endpoints

Reply:
well, the user might provide an endpoint name  while creating the endpoint so that it is easy to follow up for the user afterwards. Shall we re-create the endpoit name internally by concanating with date or expect for the user to give it unique? (I'm in favor of second option, because user might be confused, when the endpoint name does not match what they provide)